frontend/src/core/comments/components/AddComment.js

```diff
@@ -25,69 +34,87 @@ export default function AddComments(props: Props) {
         addComment,
     } = props;
 
-    const commentInput: any = React.useRef();
-    const minRows = 1;
-    const maxRows = 6;
+    let [editorState, setEditorState] = React.useState(
+        EditorState.createEmpty()
+    )
+    const editor: any = React.useRef(null);
+    const rawContentState = convertToRaw(editorState.getCurrentContent());
 
-    if (!user) {
-        return null;
+    const onChange = (editorState) => {
+        setEditorState(editorState);
     }
 
-    const handleOnChange = () => {
-        const textAreaLineHeight = 24;
-        commentInput.current.rows = minRows;
-
-        const currentRows = Math.trunc(commentInput.current.scrollHeight / textAreaLineHeight);
-
-        if (currentRows < maxRows) {
-            commentInput.current.rows = currentRows;
-        }
-        else {
-            commentInput.current.rows = maxRows;
-        }
+    function focusEditor() {
+        editor.current.focus();
     }
 
-    const handleOnKeyDown = (event: SyntheticKeyboardEvent<>) => {
-        if (event.keyCode === 13 && event.shiftKey === false) {
-            submitComment(event);
+    function keyBindingFn(e: SyntheticKeyboardEvent<>) {
```

Comment:
It seems quite complicated how you handle the keyboard shortcut for Enter. Do we need to separate methods?

Reply:
This relates to the above [comment](https://github.com/mozilla/pontoon/pull/1609#discussion_r405242400).  I can look into this further, but as I understand it, yes.

Reply:
Thanks! Please mind the indent and the colon.

Super nit: it sees more "drafty" to name the command with "kebab-case".

Reply:
To clarify the 'super nit': Do you mean change from `submitOnEnter` to `submit-on-enter`?

Reply:
Correct :)